scripts/cpplint.py

```diff
@@ -5553,6 +5567,14 @@ def CheckForIncludeWhatYouUse(filename, clean_lines, include_state, error,
       if prefix.endswith('std::') or not prefix.endswith('::'):
         required['<string>'] = (linenum, 'string')
 
+    # Ostream is special too -- also non-templatized
```

Comment:
Note that there's a number of others in iosfwd that might be useful to include. (istream being most obvious, but things like fstream, ostringstream, etc also come up somewhat frequently.) Where you fall on the completeness vs yagni scale is up to you.

Reply:
This was a specific case that I expect to be in nearly all classes because it's required for googletest to be able to print stuff.

Feel free to add others as needed.